orttraining/orttraining/training_api/utils.h

```diff
@@ -10,6 +10,28 @@
 namespace onnxruntime {
 namespace training {
 namespace api {
+
+struct ModelIdentifiers {
+  const std::string train_model;
```

Comment:
shall we have comments explaining the relation between train_model_path and train_model_data/_len? I believe either of them exists but only one of them should exists.